libraries/AC_Avoidance/AP_OABendyRuler.cpp

```diff
@@ -36,6 +76,9 @@ bool AP_OABendyRuler::update(const Location& current_loc, const Location& destin
     const float bearing_to_dest = current_loc.get_bearing_to(destination) * 0.01f;
     const float distance_to_dest = current_loc.get_distance(destination);
 
+    //make sure user has set a meaningful value for _lookahead
```

Comment:
small white space thing, normally we put a space after the "//" and the next word.  Sorry to be annoying.